### PR TITLE
feat(FRO-17):ADDED FAQ PAGE

### PR DESCRIPTION
**Pull request by preshpi**

### CODE REVIEW

File: packages/web/src/components/Faq/faq_nav.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// import React, {useState} from 'react';
```
Comment (OkoloJohnbosco) [inline packages/web/src/components/Faq/faq_nav.js:1]: Why did u comment out the whole file?? @preshpi

Comment (OkoloJohnbosco) [reply packages/web/src/components/Faq/faq_nav.js:1]: Can u also change ur naming convention to faq-data or faq-footer

**Review by OkoloJohnbosco — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: packages/web/src/components/Faq/faq_nav.js (hunk 1 above)
Comment (preshpi) [reply packages/web/src/components/Faq/faq_nav.js:1]: I thought you were going to add another navbar. I will uncomment it.

Comment (preshpi) [reply packages/web/src/components/Faq/faq_nav.js:1]: > Can u also change ur naming convention to faq-data or faq-footer

Please I don't get. Should I remain the faq_data? to faq-data?

Comment (OkoloJohnbosco) [reply packages/web/src/components/Faq/faq_nav.js]: Yes ma

Comment (OkoloJohnbosco) [reply packages/web/src/components/Faq/faq_nav.js]: For the sake of time I will do a project clean up later

**Review by OkoloJohnbosco — APPROVED** [review]
For hosting sake

**Pull request merged**